llvm/lib/Target/AArch64/AArch64FrameLowering.cpp

```diff
@@ -197,6 +197,7 @@
 #include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/Statistic.h"
 #include "llvm/CodeGen/LivePhysRegs.h"
```

Comment:
Can you remove this include?

Reply:
Does not compile when I do that

Reply:
Can you explain why? I don't see any other uses of `LivePhysRegs` in this file. What is it relying on?

Reply:
> Can you explain why? I don't see any other uses of `LivePhysRegs` in this file. What is it relying on?

computeLiveIns method.